amazons3/amazonS3_sort.bal

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+string[] alphabet = ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l", "m", "n", "o", "p", "q", "r", "s",
+"t", "u", "v", "w", "x", "y", "z"];
+string[] result = [];
+
+type SortBucket object {
+
+    string[] items = [];
+    int index = 0;
+
+
+    function sortBucket();
+
+    public function addItem(string item) {
+        self.items[self.items.length()] = item;
+    }
+};
+
+function SortBucket.sortBucket() {
```

Comment:
Move the method definition inside the object itself. Outside method definitions will be removed in 1.0